tools/content-generator/src/test/java/org/projectnessie/tools/contentgenerator/ITReadCommits.java

```diff
@@ -69,23 +70,22 @@ void readCommits() throws UnsupportedEncodingException, NessieNotFoundException
   @Test
   void readCommitsVerbose() throws UnsupportedEncodingException, NessieNotFoundException {
     ByteArrayOutputStream baos = new ByteArrayOutputStream();
-    PrintStream out = new PrintStream(baos);
-    System.setOut(out);
-
-    assertThat(
-            NessieContentGenerator.runMain(
-                new String[] {
-                  "commits", "--uri", NESSIE_API_URI, "--ref", branch.getName(), "--verbose"
-                }))
-        .isEqualTo(0);
+    try (PrintWriter out = new PrintWriter(new PrintStream(baos), true)) {
+      assertThat(
+              NessieContentGenerator.runMain(
+                  out,
+                  new String[] {
+                    "commits", "--uri", NESSIE_API_URI, "--ref", branch.getName(), "--verbose"
+                  }))
+          .isEqualTo(0);
 
-    out.close();
-    String[] output = baos.toString(StandardCharsets.UTF_8.toString()).split("\n");
-    assertThat(output).anySatisfy(s -> assertThat(s).contains("testMessage"));
-    assertThat(output).anySatisfy(s -> assertThat(s).contains("first.second"));
-    assertThat(output).anySatisfy(s -> assertThat(s).contains("key[0]: first"));
-    assertThat(output).anySatisfy(s -> assertThat(s).contains("key[1]: second"));
-    assertThat(output).anySatisfy(s -> assertThat(s).contains(contentId));
+      String[] output = baos.toString(StandardCharsets.UTF_8.toString()).split("\n");
+      assertThat(output).anySatisfy(s -> assertThat(s).contains("testMessage"));
+      assertThat(output).anySatisfy(s -> assertThat(s).contains("first.second"));
+      assertThat(output).anySatisfy(s -> assertThat(s).contains("key[0]: first"));
+      assertThat(output).anySatisfy(s -> assertThat(s).contains("key[1]: second"));
+      assertThat(output).anySatisfy(s -> assertThat(s).contains(contentId));
```

Comment:
nit: slightly odd to split output by lines... only to the then confirm ANY line contains the given output